apps/web/src/pages/Pools/AssessmentPlanBuilderPage/components/OrganizeSection.tsx

```diff
@@ -0,0 +1,177 @@
+                },
+              )}
+            </Accordion.Trigger>
+          </Accordion.Header>
+          <Accordion.Content>
+            <p>
+              {intl.formatMessage(
+                {
+                  defaultMessage:
+                    'Ultimately, it\'s the decision of the hiring panel or manager to determine how many assessment methods are appropriate. If you have more than {maxSteps}, select "Additional Assessment" for your last choice, and then reach out to our team. We can walk you through how these additional assessments can be handled on the platform.',
+                  id: "9IMcqY",
+                  description:
+                    "First paragraph of second answer of the Frequently Asked Questions for logging in",
+                },
+                {
+                  maxSteps: MAX_ASSESSMENT_STEPS,
+                },
+              )}
+            </p>
+          </Accordion.Content>
+        </Accordion.Item>
+      </Accordion.Root>
+      <div data-h2-margin-top="base(x1)">
+        {/* TODO: replace with repeater cards */}
+        {pool.assessmentSteps
+          ?.filter(notEmpty)
+          .sort((a: AssessmentStep, b: AssessmentStep) =>
+            (a.sortOrder ?? 0) > (b.sortOrder ?? 0) ? 1 : -1,
+          )
+          .map((step: AssessmentStep) => (
+            <div key={step.id}>
+              <p>{step.type}</p>
+              <p>{getLocalizedName(step.title, intl)}</p>
+            </div>
+          ))}
+        {/* TODO: Adding some strings for translation that will be needed later */}
+        {intl.formatMessage({
+          defaultMessage: "Add a new assessment",
+          id: "VWLWLY",
+          description:
+            "Button text to add a new assessment to the assessment plan",
+        })}
+        {intl.formatMessage({
+          defaultMessage: "You have too few assessments",
+          id: "ypxaI2",
+          description:
+            "Title for warning message when the user has few assessments to the assessment plan",
+        })}
+        {intl.formatMessage({
+          defaultMessage:
+            "Consider adding at least one assessment method to strengthen the quality of your candidate pool.",
+          id: "xCoGIm",
+          description:
+            "Description for warning message when the user has few assessments to the assessment plan",
+        })}
+        {intl.formatMessage({
+          defaultMessage: "You are approaching the limit!",
+          id: "1moJ8r",
+          description:
+            "Title for warning message when the user has added many assessments to the assessment plan",
+        })}
+        {intl.formatMessage(
+          {
+            defaultMessage:
+              "You can add up to {maxSteps} assessment methods, but you don’t need to. You are covered as long as every skill has been taken into account. Choosing too many assessment methods can delay the staffing process and reduce the chances of a successful hire.",
```

Comment:
Smart apostrophe on "don't" 🤓

Reply:
Nice catch! [get rid of smart apostrophes](https://github.com/GCTC-NTGC/gc-digital-talent/pull/8225/commits/c36c915fdbdfda63a89042cb100959aed553ab6c)